---
layout: page
title:  High-throughput Identification of DNA Repair Products
description: a project that we developed a series of softwares to adress DNA repair products at DSB sites. 
img: assets/img/7.jpg
importance: 3
category: work
related_publications: no
---

**Development of iDSBins**

Insertions of mobile elements, mitochondrial DNA and fragments of nuclear chromosomes at DNA double-strand breaks sites (DSBs) threaten genome integrity and are common in cancer. Despite extensive efforts, our knowledge of these insertions still remains unknown. These large insertions were previously profiled along with quantitatively Sanger sequencing, but have not yet been combined with the massively parallel approaches to tackle the complexity of insertion events. Here, we introduced a high-throughput sequencing Break-Ins (DNA double strand break insertion sequencing), a method that can detect large and complicated insertion/deletion events at DNA double strand break sites. In parallel with the novel technology, we also developed a simplified, standardized and fully automated data analysis software toolkit, iDSBins, which enables routine scoring and interpretation of large-scale insertions at targeted double strand break site.

[iDSBins Github](https://github.com/gucascau/iDSBins.git)

**Development of LargeInsertionFeature**

Insertions of mobile elements, mitochondrial DNA and fragments of nuclear chromosomes at DNA double-strand breaks sites (DSBs) threaten genome integrity and are common in cancer. Despite extensive efforts, our knowledge of these insertions still remains unknown. These tutorials will focus on the different features of large insertion events.

[LargeInsertionFeature Github](https://github.com/gucascau/LargeInsertionFeature.git)

**Development of iDSBInvert**

Templated insertions can occur at programmed breaks at VDJ locus or at endonuclease induced break. Despite their prevalence, the mechanisms underlying such events remain poorly understood, and the genetic pathways regulating these events are largely unknown. We propose that such inversions arise through microhomology mediated template switch within single-stranded DNA intermediates of replication, followed by displacement of inverted sequence and its insertion at the DSB. The package is to characterize the inverted repeats from ssDNA transformation to undestand the mechanism.

[iDSBInvert Github](https://github.com/gucascau/iDSBInvert.git)

**Development of iDSBindel**

DNA double strand breaks are one of the most deleterious DNA lesions. The repair by nonhomologous end joining is error-prone, which would induced small insertions or remove damaged or mismatched nucleotides. To detect the products of DNA double strand break repair, we developed a novel techonology -- Break-Ins. In accompany with this technology, we also developed iDSBindel, which would sensentively detect short insertion, short deletion, short deletion and short insertion, and indel-free products at DNA double strans break site. iDSBindel can also calculate the frequency of each indel repair events.

[iDSBindel Github](https://github.com/gucascau/iDSBindel.git)
